khelo/lib/ui/flow/stats/user_stat/user_stat_view_model.dart

```diff
@@ -52,6 +53,109 @@ class UserStatViewNotifier extends StateNotifier<UserStatViewState> {
     }
   }
 
+  UserStat _calculateUserStats(List<BallScoreModel> ballList) {
```

Comment:
is it possible to break down this function to smaller functions to reduce it's size?